### ISSUE TITLE
[4.0.0] indent + multiline expression not working as expected

### ISSUE THREAD

**Issue by Trott** [eslint/eslint#8721]
**Tell us about your environment**

* **ESLint Version:** 4.0.0
* **Node Version:** 8.1.0
* **npm Version:** 5.0.3

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

```
rules:
  indent: [2, 2, {ArrayExpression: first,
                  CallExpression: {arguments: first},
                  FunctionDeclaration: {parameters: first},
                  FunctionExpression: {parameters: first},
                  MemberExpression: off,
                  ObjectExpression: first,
                  SwitchCase: 1}]
```

**What did you do? Please include the actual source code causing the issue.**

```js
var foo = bar(function() {
  doSomething();
}).baz(function() {
  doSomethingElse();
});
```

**What did you expect to happen?** No errors.

**What actually happened? Please include the actual, raw output from ESLint.**

```console
$ eslint foo.js

/Users/trott/test/foo.js
  4:1  error  Expected indentation of 4 spaces but found 2  indent
  5:1  error  Expected indentation of 2 spaces but found 0  indent

✖ 2 problems (2 errors, 0 warnings)
  2 errors, 0 warnings potentially fixable with the `--fix` option.

$
```

I'm not sure if this has the same root cause as https://github.com/eslint/eslint/issues/8717 or if it is distinct...

**Comment by not-an-aardvark** [comment]
Thanks, I can reproduce this. It seems to be a bug with the `MemberExpression: off` option.
